### PR TITLE
Update dynamic framework installation instructions.

### LINKED ISSUE #609: Simplify SDK 3.0 Installation instructions
```
Dynamic Framework Installation ⤴

Add Buy.framework target as a dependency:
Open Your Project, and then select your target.
Navigate to Build Phases > Target Dependencies.
Add Buy.framework.
Link Buy.framework:
Open Your Project, and then select your target.
Navigate to Build Phases > Link Binary With Libraries.
Add Buy.framework.
Make sure that the framework is copied into the bundle:
Open Your Project, and then select your target.
Navigate to Build Phases > New Copy Files Phase.
From the Destination dropdown, select Frameworks.
Add Buy.framework.
Import into your project files using import Buy.
```

This might be a markdown formatting issue on github, but a lot of steps seem redundant.

@dbart01

### LINKED ISSUE #624: Swift Sample Application Fails to Build: No such file - GraphQL.swift
Trying to get the Storefront sample application working, but it looks like the GraphQL.swift file is not included in the repo.  

I've downloaded the application, added the Mobile Sales Channel, and modified the Client.swift file, but xCode says I'm missing a file. 

```
<unknown>:0: error: no such file or directory: '/Users/bpluim/Code/Sandbox/mobile-buy-sdk-ios/Dependencies/Swift Gen/support/Sources/GraphQL.swift'

Command /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc failed with exit code 1
```

### PR DESCRIPTION
**Pull request by dbart01**
Update the README instructions and fix indentation.

Fixes #609 
Fixes #624

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,46 +83,43 @@ The documentation is generated using [Jazzy](https://github.com/realm/jazzy).
...
+  1. Navigate to **Build Phases** > **Link Binary With Libraries**.
+  2. Add `Buy.framework`.
 3. Make sure that the framework is copied into the bundle:
-  1. Open **Your Project**, and then select your target.
-  2. Navigate to **Build Phases** > **New Copy Files Phase**.
-  3. From the **Destination** dropdown, select **Frameworks**.
-  4. Add `Buy.framework`.
+  1. Navigate to **Build Phases** > **New Copy Files Phase**.
+  2. From the **Destination** dropdown, select **Frameworks**.
+  3. Add `Buy.framework`.
```
Comment (davidmuzi) [inline README.md]: this still appears as a flatlets 1-11 in github markdown

Comment (dbart01) [inline README.md]: Weird, shows up differently in the MacDown editor. Fixed.

File: README.md [hunk 2]
```diff
@@ -83,46 +83,43 @@ The documentation is generated using [Jazzy](https://github.com/realm/jazzy).
 ### Dynamic Framework Installation [⤴](#table-of-contents)
 
 1. Add **Buy.framework** target as a dependency:
-  1. Open **Your Project**, and then select your target.
-  2. Navigate to **Build Phases** > **Target Dependencies**.
-  3. Add `Buy.framework`.
+    1. Navigate to **Build Phases** > **Target Dependencies**.
```
Comment (davidmuzi) [inline README.md]: do you need to include a step to include the Buy.xcodeproj in the users project? Its not apparent here

Comment (dbart01) [inline README.md]: I've updated the instructions to include all the steps necessary to setup the dynamic framework.

Comment (dbart01) [inline README.md]: It also addresses some concerns brought up in #624

**Pull request merged**